devices.py

```diff
@@ -59,35 +61,296 @@ class Newport(EpicsMotor):
     """
     Basic class for the newport motors.
     """
-    home_forward = C(Signal)
-    home_reverse = C(Signal)
-    offset_freeze_switch = C(Signal)
+
+    home_forward = Cpt(Signal)
+    home_reverse = Cpt(Signal)
+    offset_freeze_switch = Cpt(Signal)
 
     #@retry(tries=3)
     def move(self, *args, **kwargs):
         return super().move(*args, **kwargs)
 
 
-class SeqBase(SndDevice):
+class Sequencer(Device):
     """
     Base sequencer class.
+    
+    Attributes
```

Comment:
I have in previous `ophyd` classes listed the components but I'm starting to change my mind on it. You can continue doing it if you feel it's useful, but don't feel obligated to just because you see them around.